### PR TITLE
CB-3268 feat: admin users pagination

### PR DESCRIPTION
**Pull request by Wroud**

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java [hunk 1]
```diff
@@ -74,7 +74,10 @@ void createUser(
     SMUser getUserById(String userId) throws DBException;
 
     @NotNull
-    SMUser[] findUsers(String userNameMask) throws DBException;
+    SMUser[] findUsers(@NotNull Integer first, @Nullable String after, @NotNull String userIdMask, Boolean enabledState)
```
Comment (alexander-skoblikov) [inline plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java]: What is `first`? What is `after`?
Why `first` is Integer, but `after` is string? Do you mean limit/offset?

File: plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java [hunk 2]
```diff
@@ -74,7 +74,10 @@ void createUser(
...
+            throws DBException;
+
+    int countUsers(@Nullable String after, @NotNull String userIdMask, Boolean enabledState) throws DBException;
```
Comment (alexander-skoblikov) [inline plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java]: is this method really needed in the interface?

Comment (alexander-skoblikov) [reply plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java]: it is better to move the 'user filter' in a separate class and use one model to count / search users

File: plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java (hunk 1 above)
Comment (Wroud) [reply plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java]: replaced with limit/offset

File: plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java (hunk 2 above)
Comment (Wroud) [reply plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java]: filters moved to separate class

Comment (Wroud) [reply plugins/org.jkiss.dbeaver.model.sm/src/org/jkiss/dbeaver/model/security/SMAdminController.java]: this method is not in use right now but it is required for pagination (to calculate pages)

**Pull request merged**